### ISSUE TITLE
{"success":false,"error":{"code":0,"message":"The web service did not return alerts count.","trace":""}}

### ISSUE THREAD

**Issue by hobejeans** [YetiForceCompany/YetiForceCRM#8739]
#### How to trigger the error

1.
2.
3.

#### Your Environment

* YetiForce Version used:
* Browser name and version:
* Environment name and version:
* Operating System and version:

**Comment by hobejeans** [comment]
How do I fix this error on the user preference home tab & system warning.
I deleted and reinstalled the crm & the same error is there. It appeared a few days after the first install.

{"success":false,"error":{"code":0,"message":"The web service did not return alerts count.","trace":""}}

**Comment by mariuszkrzaczkowski** [comment]
This occurs during verification of vulnerabilities in dependent libraries if an error occurs.

SensioLabs has been recently acquired by Symfony and a lot of changes are being made to their product, e.g., limits were added on the connection to their server and this may be the reason for this error message.

In the current developer version, we updated their library and secured everything against such problems, but this will appear with the new stable release.

The simplest solution is to disable the verification of SecurityChecker mechanism and wait for a new release. 

To disable simply delete the file app/SystemWarnings/Security/Dependencies.php
